apps/institution-service/src/tasks/loadInstitutionsFromJson.ts

```diff
@@ -96,14 +101,29 @@ async function loadInstitutionData() {
         isActive: false,
         aggregatorId: finicityAggregator?.id,
         aggregator_institution_id: finicity.id,
-        institution_id: institution.ucp_id,
+        institution_id: institution.id,
         supports_oauth: finicity.supports_oauth,
         supports_identification: finicity.supports_identification,
         supports_verification: finicity.supports_verification,
         supports_history: finicity.supports_history,
         supports_aggregation: finicity.supports_aggregation,
       } as AggregatorIntegration);
     }
+
+    if (institution?.testExampleA?.id) {
+      const testExampleA = institution.testExampleA;
+      aggregatorList.push({
+        isActive: false,
```

Comment:
I think we want the actual isActive value here

Reply:
Good catch

Reply:
I just realized the json file doesn't include the isActive field, should we add that?

Reply:
I don't think ucw-app uses it at all so it's probably not necessary to include in the cacheList endpoint right?

Reply:
No we don't need to return it in the cacheList endpoint, because we don't return institutions that aren't active in the cacheList.

I put isActive to false on Finicity because they aren't active, and so that I could use that for tests. I think ideally the institution map file has isActive, so we can run tests from a solid base and test institutions that aren't active.